package.json

```diff
@@ -82,9 +81,9 @@
   },
   "lint-staged": {
     "*.{js,flow}": [
-      "lint",
-      "test",
-      "flow"
+      "yarn lint",
+      "yarn flow focus-check",
```

Comment:
Oh I didn't know this command, what's the difference with `check`

Reply:
I had some issues with `lint-staged` library and based on their documentation, they recommended using `focus-check` ([link](https://github.com/okonet/lint-staged#typecheck-your-staged-files-with-flow)).

`focus-check` explanation: 
```EXPERIMENTAL: Does a focused Flow check on the input files/directories (and each of their dependents and dependencies) and prints the results.```